extensions/blocks/podcast-player/edit.js

```diff
@@ -326,12 +329,13 @@ const PodcastPlayerEdit = ( {
 					title={ feedData.title }
 					link={ feedData.link }
 				/>
-				{
-					// Disabled because the overlay div doesn't actually have a role or functionality
-					// as far as the user is concerned. We're just catching the first click so that
-					// the block can be selected without interacting with the embed preview that the overlay covers.
-					/* eslint-disable jsx-a11y/no-static-element-interactions */
-				 }
+				{ /*
+				 * Disabled because the overlay div doesn't actually have a role or
```

Comment:
The indentation doesn't seem to match

Reply:
This is how Prettier wants it currently. It will probably be improved at some point but now it automatically formats like this. Unless we want to use another format here? Personally I'm fine with this one.